stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -21,6 +21,7 @@
 module SwiftGlibc [system] {
 % if CMAKE_SDK in ["LINUX", "FREEBSD", "CYGWIN"]:
   link "pthread"
+  link "util"
```

Comment:
It is a limitation of the current Clang modules implementation, unfortunately. My understanding it is non-trivial to fix that.